docs/reference/api/docker_remote_api_v1.25.md

```diff
@@ -4401,6 +4401,42 @@ Content-Type: text/plain; charset=utf-8
 -   **404** - plugin not installed
 -   **500** - plugin is active
 
+### Create a plugin
+
+`POST /plugins/(plugin name)/create`
```

Comment:
@anusha-ragunathan it's fine for enable disable, it's an action on an existing resource. (container start and stop are the same)

but I see @thaJeztah's point, it's different for create, the resource doesn't yet exist. (same for image or container create, it's not in the URL)